### ISSUE TITLE
PopUp hidden behind component

### ISSUE THREAD

**Issue by Nairolf92** [jacklam718/react-native-popup-dialog#56]
Hello, 

It seems that the pop-up can't be shown when it is inside another component. Looks like a z-index problem like in CSS.

I'm using **react-native-calendars** , and inside the `<Agenda>` component I am rendering multiple items. 
Here is a portion of my code where the `<PopupDialog>` component is present.

```
 renderItem(item) {
    return (
      <View style={[styles.item]}>
        <Grid>
          <Col size={65}>
            <Text style={[styles.hour]}>{item.hour}</Text>
            <Text style={[styles.name]}>{item.name}</Text>
            <Text style={[styles.prestation]}>{item.prestation}</Text>
          </Col>
          <Col size={35} style={{paddingTop : 10}}>
            <Button rounded info style={StyleSheet.flatten(styles.badge)}
            onPress={() => {this.popupDialog.show();}}>
              <Icon active name='search'/>
              <Text Détails</Text>
            </Button>
          </Col>
        </Grid>
        <PopupDialog
          ref={(popupDialog) => { this.popupDialog = popupDialog; }}
        >
          <View>
            <Text>Hello</Text>
          </View>
        </PopupDialog>
      </View>
    );
  }
```

**Comment by Raktima** [comment]
Same issue

**Comment by jacklam718** [comment]
@Nairolf92 @Raktima The issue has been fixed in `v0.9.30`
